upgrades/upgradevalidation/validation.go

```diff
@@ -151,24 +153,50 @@ var windowsSeries = []string{
 }
```

Comment:
not sure how windowsSeries looks like this. 
https://github.com/juju/juju/blob/2.9/upgrades/upgradevalidation/validation.go#L153

Reply:
3.0 has deleted the Windows series from core, so needs to be done manually here